### PR TITLE
Update to x25519-key-pair v2.0.0 (changed fingerprint prefix).

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Addresses issue #15.

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
What the devil.. the tests pass locally. (As in, my Node 12 env is coming up with a different fingerprint than Travis's).

**Comment by davidlehn** [comment]
I get same results as travis.

**Comment by dmitrizagidulin** [comment]
(k, Travis passing now.)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # did:key driver ChangeLog
 
+## 0.6.0 - 2020-03-09
+
+### Changed
+- **BREAKING**: Use `x25519-key-pair` v2.0.0, changed fingerprint format
+  for X25519 keys.
+- Use `crypto-ld` v0.3.7.
```
Comment (davidlehn) [inline CHANGELOG.md:8]: Was this supposed to be 3.7.0?

Comment (dmitrizagidulin) [reply CHANGELOG.md:8]: @davidlehn whoops, yes :) thank you for the catch.

Comment (dmitrizagidulin) [reply CHANGELOG.md:8]: (The actual `package.json` has the correct version.)

Comment (davidlehn) [reply CHANGELOG.md:8]: Yup, no big deal.  Just needs a quick CHANGELOG update.

**Pull request merged**